yaps2/pipelines/postvqsr38.py

```diff
@@ -490,6 +492,37 @@ def create_remove_ac_0_tasks(self):
 
         return tasks
 
+    def create_remove_symbolic_deletion_tasks(self):
+        tasks = []
+        stage = '4-remove-symbolic-alleles'
+        basedir = os.path.join(self.config.rootdir, stage)
+
+        lsf_params = get_lsf_params(
+                remove_symbolic_deletion_alleles_lsf_params,
+                self.config.email,
+                self.config.docker
+                )
+        lsf_params_json = to_json(lsf_params)
+
+        for chrom in self.config.chroms:
+            vcf = self.config.vcfs[chrom]
+            output_vcf = 'combined.c{chrom}.vcf.gz'.format(chrom=chrom)
```

Comment:
@indraniel - Is this the right filename here? It seemed like you re-used this in a couple of places.